src/main/resources/messages_es_ES.properties

```diff
@@ -434,15 +434,15 @@ addPageNumbers.selectText.4=Número de inicio
 addPageNumbers.selectText.5=Páginas a numerar
 addPageNumbers.selectText.6=Texto personalizado
 addPageNumbers.customTextDesc=Texto personalizado
-addPageNumbers.numberPagesDesc=Qué páginas numerar, por defecto 'all', también acepta 1-5 o 2,5,9 etc

```

Comment:
These are key words that the code looks for, you can't change the all tag or the filename tag etc